packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -1,189 +1,208 @@
 @use '../layout' as *;
 @use '../mixins';
 @use '../base' as *;
+@use '../utilities/grid' as *;
+@use '../utilities/display-visibility' as *;
 
 .ds-c-usa-banner {
   background-color: var(--usa-banner__background-color);
-  color: var(--usa-banner__color);
-  padding: 0;
+  color: var(--usa-banner-color);
 }
 
 .ds-c-usa-banner__header {
-  align-items: flex-start;
-  display: flex;
+  // margin-inline: auto;
+  // max-width: var(--usa-banner__max-width);
+  // padding-inline: calc(var(--usa-banner-gutter-width) * 0.5);
+  @extend .ds-l-container;
```

Comment:
I would like to remove these `@extend` rules if possible. I tried creating custom props where teams could overwrite the banner's max-width if needed, but they aren't generating correctly 😢